lib/jsonapi/serializer.ex

```diff
@@ -79,15 +79,13 @@ defmodule JSONAPI.Serializer do
   @spec encode_relationships(Conn.t(), document(), tuple(), list()) :: tuple()
   def encode_relationships(conn, doc, {view, data, _, _} = view_info, options) do
     view.relationships()
-    |> Enum.filter(&data_loaded?(Map.get(data, get_data_key(&1))))
```

Comment:
Discussion Point: Do we want to put the new functionality behind some sort of flag?

```elixir
if options[:auto_include_relationship_ids]
    view.relationships()
    |> Enum.map_reduce(doc, &build_relationships(data, conn, view_info, &1, &2, options))
else
    view.relationships()
    |> Enum.filter(&data_loaded?(Map.get(data, get_data_key(&1))))
    |> Enum.map_reduce(doc, &build_relationships(data, conn, view_info, &1, &2, options))
end

```